students/BrianB/lesson04/mailroom2.py

```diff
@@ -5,23 +5,46 @@
 # Who: Brian Brumleve
```

Comment:
It looks like you ran into a merge issue, instead of commenting out the changes that you don't want, you should just remove them from the file. Otherwise, this adds a bunch of extra commented out junk in your file which can affect readability and provides nothing